srcpkgs/stremio-shell/template

```diff
@@ -35,21 +35,27 @@ post_extract() {
 }
 
 post_install() {
+
 	# Fix the bizarre locations
 	vmkdir usr/lib/stremio
-	mv "${DESTDIR}"/usr/opt/stremio/stremio ${DESTDIR}/usr/lib/stremio
+	mv "${DESTDIR}/usr/opt/stremio/stremio" "${DESTDIR}/usr/lib/stremio"
 
+	# Copy .desktop file
```

Comment:
Please don't add misleading comments.

Reply:
I will make it less misleading - move. Thanks.